src/main/src/main/java/org/geoserver/catalog/impl/LayerGroupInfoImpl.java

```diff
@@ -29,6 +29,7 @@
     protected String id;
     protected String name;
     protected Mode mode = Mode.SINGLE;
+    protected Boolean forceNotQueryable = Boolean.FALSE;
```

Comment:
Just "queryable" like in LayerInfo (and you'll have to make opposite logic to handle null values, layer group is queryable by default).